config/link-check/excluded-files.yml

```diff
@@ -0,0 +1,4 @@
+- 'src/consts.js'
```

Comment:
should we make part of the config for simplicity?

Reply:
though, I think it makes sense to keep it separate, since this list might become large

Reply:
I'd say it's pretty safe to put the file patterns in the main config if needed, as with the pattern match feature on both whitelists and blacklists we have quite a bit of control over the files checked with only a few entries.

That said, keeping them separate is at least a bit more future-proof and keeps consistent with the link filters.